app/[locale]/director-message/page.tsx

```diff
@@ -10,55 +71,212 @@ export default async function DirectorCorner({
-          <p className="w-fit max-w-4xl text-justify text-lg max-md:rounded-t md:rounded-r">
-            <span className="bold mb-5 block">{text.message[11]}</span>
-          </p>
-        </article>
-        <article className="flex-end flex justify-end">
-          <p className="w-fit max-w-4xl text-justify text-lg font-bold max-md:rounded-t md:rounded-r">
-            <span className="bold mb-5 block">{text.message[12]}</span>
-          </p>
-        </article>
-      </section>
+          </div>
+        </section>
+      </div>
+      {/* For DIRECTOR's MESSAGE */}
+      <div
+        className="w-full bg-cover bg-center"
+        style={{
+          backgroundImage: `linear-gradient(rgba(249, 245, 235, 0.95) 0%, rgba(249, 245, 235, 0.85) 85%, rgba(249, 245, 235, 0.95) 100%), url('${getS3Url()}/assets/temple-1.jpeg')`,
+        }}
+      >
+        <section className="container mx-auto">
+          <Heading
+            text={text.title.toUpperCase()}
+            heading="h3"
+            glyphDirection="dual"
+          />
+          <article className="w-full">
+            <p className="text-lg max-md:rounded-t md:w-full md:rounded-r ">
+              {text.message.slice(0, 7).map((message, index) => (
+                <span key={index} className="mb-5 block">
+                  {message}
+                </span>
+              ))}
+            </p>
+          </article>
+          <article className="mx-auto flex items-center justify-center max-md:flex-col">
+            <p className="w-fit max-w-4xl text-center text-lg max-md:rounded-t md:rounded-r">
+              {text.message.slice(7, 9).map((message, index) => (
+                <span key={index} className="mb-5 block">
+                  {message}
+                </span>
+              ))}
+            </p>
+          </article>
+          <article className="flex max-md:flex-col">
+            <p className="text-lg max-md:rounded-t md:w-full md:rounded-r ">
+              {text.message.slice(9, 11).map((message, index) => (
+                <span key={index} className="mb-3 block">
+                  {message}
+                </span>
+              ))}
+            </p>
+          </article>
+          <article className="mx-auto flex items-center justify-center max-md:flex-col">
+            <p className="w-fit max-w-4xl text-justify text-lg max-md:rounded-t md:rounded-r">
+              <span className="bold mb-5 block">{text.message[11]}</span>
+            </p>
+          </article>
+          <article className="flex-end flex justify-end">
+            <p className="w-fit max-w-4xl text-justify text-lg font-bold max-md:rounded-t md:rounded-r">
+              <span className="bold mb-5 block">{text.message[12]}</span>
+            </p>
+          </article>
+        </section>
+      </div>
     </>
   );
 }
+
+interface ContactDetail {
+  label: string;
+  icon?: React.ReactNode | string;
+}
+
+interface CardProps {
```

Comment:
`CardProps` is only used once in the file, there's no use abstracting it out to a reusable interface. Same with `ContactDetail` above this.